App.js

```diff
@@ -27,6 +37,17 @@ export default class App extends Component {
       error => console.log(error),
       { enableHighAccuracy: false, timeout: 20000, maximumAge: 0, distanceFilter: 1 }
     );
+  }
+
+  componentDidMount = async () => {
+    const { Permissions } = Expo;
+    const { status } = await Permissions.askAsync(Permissions.LOCATION);
```

Comment:
We are just getting permission for location here, we we have to add one for camera too if we add that feature

Reply:
Ok, sounds good